superblock_construction_protocol.md

```diff
@@ -1,5 +1,8 @@
 # Superblock Construction Protocol (SBCP) <!-- omit from toc -->
 
+> [!WARN]
+> This version is deprecated. Please refer to the [SBCP v2 document](./superblock_construction_protocol_v2.md) instead.
+
```

Comment:
I think just move to `v1` folder...
Or we can delete and just rely on git tags.

We need to think on the best way with @moshe-blox and @ayaz-ssvlabs. Once the network goes live and we have such updates they will have to support both versions.